src/storage/storage.cc

```diff
@@ -235,6 +235,7 @@ Status Storage::Open(bool read_only) {
   db_closing_ = false;
 
   bool cache_index_and_filter_blocks = config_->rocks_db.cache_index_and_filter_blocks;
+  size_t block_cache_size = config_->rocks_db.block_cache_size * MiB;
```

Comment:
Correct the block_cache_size value to metadata_block_cache_size + subkey_block_cache_size if it's zero.